lisp/mastodon-tl.el

```diff
@@ -197,8 +196,8 @@ also render the html"
   (let ((content (mastodon-tl--field 'content toot)))
     (propertize (with-temp-buffer
                   (insert (decode-coding-string content 'utf-8))
-                  (customize-set-value 'shr-use-fonts nil)
-                  (shr-render-region (point-min) (point-max))
+                  (let ((shr-use-fonts nil))
```

Comment:
No need to call `let` again since it happens earlier in the function.

```elisp
(let ((content (mastodon-tl--field 'content toot))
      (shr-use-fonts nil))
  ;; ...
```